packages/app-store/jitsivideo/zod.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from "zod";
+
+export const appKeysSchema = z.object({
+  jitsi_host: z.string().default("https://meet.jit.si"),
```

Comment:
```suggestion
  jitsi_host: z.string().default("https://meet.jit.si/cal/"),
```

The default is this. Also, / is very important at the end, otherwise the link would be broken.